src/JWT/Builder/JwtBuilder.cs

```diff
@@ -296,6 +297,17 @@ public T Decode<T>(string token)
             return _decoder.DecodeToObject<T>(token, _secrets, _valParams.ValidateSignature);
         }
 
+        private static IJsonSerializer CreateSerializer()

```

Comment:
Should we move this factory method to `JsonSerializerFactory`? Right now it's present in every class that uses `IJsonSerializer`.

Reply:
Yeah, that would be nice. I'm going to bed now, pushing a little fix for unittest on linux, hopefully it will work on windows too :)